app/src/main/res/values/strings.xml

```diff
@@ -414,6 +415,8 @@ https://github.com/RocketChat/java-code-styles/blob/master/CODING_STYLE.md#strin
     <string name="close_button_description">Close</string>
     <string name="go_back_button_description">Go back</string>
     <string name="menu_button_description">Menu</string>
+    <string name="leave">Leave</string> <!-- TODO Add translation -->
+    <string name="left_chat">You left the chat</string> <!-- TODO Add translation -->
```

Comment:
We're going to need to put in the spanish translation for these as the bulk of our initial users are in mexico